lib/validators/mbtiles.js

```diff
@@ -94,16 +98,61 @@ function zoomCheck(db, limits, callback) {
   
   log.debug('zoomCheck(): checking...');
   queue()
-    .defer(getMaxZoom, 'tiles')
-    .defer(getMaxZoom, 'grids')
+    .defer(getMaxZoom, 'tiles_max')
+    .defer(getMaxZoom, 'grids_max')
+    .defer(getMaxZoom, 'tiles_min')
+    .defer(getMaxZoom, 'maxzoom')
+    .defer(getMaxZoom, 'fillzoom')
     .awaitAll(function(err, zoom) {
       err = sqliteError(err);
       if (err) return callback(err);
 
-      if (zoom[0].z > limits.max_zoomlevel)
-        return callback(invalid('Maxzoom exceeded for mbtiles file. There is a max zoom limit of ' + limits.max_zoomlevel + ' but tiles up to zoom level ' + zoom[0].z + ' were found. If you need support for 1cm tilesets please reach out to support.'));
-      if (zoom[1].z > limits.max_zoomlevel)
-        return callback(invalid('Maxzoom exceeded for mbtiles file. There is a max zoom limit of ' + limits.max_zoomlevel + ' but tiles up to zoom level ' + zoom[1].z + ' were found. If you need support for 1cm tilesets please reach out to support.'));
+      var tiles_max = zoom[0];
+      var grids_max = zoom[1];
+      var tiles_min = zoom[2];
+      var maxzoom = zoom[3];
+      var fillzoom = zoom[4];
+
+      if (tiles_max.z > limits.max_zoomlevel)
+        return callback(invalid('Maxzoom exceeded for mbtiles file. There is a max zoom limit of ' + limits.max_zoomlevel + ' but tiles up to zoom level ' + tiles_max.z + ' were found. If you need support for 1cm tilesets please reach out to support.'));
+      if (grids_max.z > limits.max_zoomlevel)
+        return callback(invalid('Maxzoom exceeded for mbtiles file. There is a max zoom limit of ' + limits.max_zoomlevel + ' but tiles up to zoom level ' + grids_max.z + ' were found. If you need support for 1cm tilesets please reach out to support.'));
+      if (tiles_min.z < 0)
+        return callback(invalid('Minzoom exceeded for mbtiles file. There is a min zoom limit of 0 but tiles down to zoom level ' + tiles_min.z + ' were found.'));
+
+      if ('value' in fillzoom) {
+        if (!fillzoom.value.match('^[0-9]+$')) {
+          return callback(invalid('Fillzoom in mbtiles metadata is not numeric. Found: ' + fillzoom.value));
+        }
+        var fz = Number.parseInt(fillzoom.value, 10);
```

Comment:
Could we call this `fillZoomValue` or something like that?

Reply:
The row in the `metadata` table is called `fillzoom` so I think it would be confusing to refer to it by a different name. I will try to make the message clearer.